### PR TITLE
Added a --no-splash option for disabling the ES logo

### PR DESCRIPTION
**Pull request by jrassa**
no splash option for ES from Herdinger's fork

### CODE REVIEW

File: es-app/src/main.cpp [hunk 1]
```diff
@@ -55,7 +55,10 @@ bool parseArgs(int argc, char* argv[], unsigned int* width, unsigned int* height
 		}else if(strcmp(argv[i], "--no-exit") == 0)
 		{
 			Settings::getInstance()->setBool("ShowExit", false);
-		}else if(strcmp(argv[i], "--debug") == 0)
+		}else if(strcmp(argv[i], "--no-splash") == 0)
+        {
+            Settings::getInstance()->setBool("SplashScreen", false);
+        }else if(strcmp(argv[i], "--debug") == 0)
 		{
```
Comment (joolswills) [inline es-app/src/main.cpp]: wrong indentation

File: es-app/src/main.cpp [hunk 2]
```diff
@@ -225,8 +229,8 @@ int main(int argc, char* argv[])
 		std::string glExts = (const char*)glGetString(GL_EXTENSIONS);
 		LOG(LogInfo) << "Checking available OpenGL extensions...";
 		LOG(LogInfo) << " ARB_texture_non_power_of_two: " << (glExts.find("ARB_texture_non_power_of_two") != std::string::npos ? "ok" : "MISSING");
-
-		window.renderLoadingScreen();
+        if(Settings::getInstance()->getBool("SplashScreen"))
+            window.renderLoadingScreen();
```
Comment (joolswills) [inline es-app/src/main.cpp]: as above.

**Comment by joolswills** [comment]
Please can you squash it and I will merge. thanks.

**Comment by jrassa** [comment]
The commits have been squashed.

**Pull request merged**